business/apps.go

```diff
@@ -21,27 +21,6 @@ type AppService struct {
 	k8s  kubernetes.IstioClientInterface
 }
 
-// Temporal map of Workloads group by app label
-type appsWorkload map[string][]*models.Workload
-
-// Helper method to build a map of workloads for a given labelSelector
-func (in *AppService) fetchWorkloadsPerApp(namespace, labelSelector string) (appsWorkload, error) {
-	cfg := config.Get()
-
-	ws, err := fetchWorkloads(in.k8s, namespace, labelSelector)
-	if err != nil {
-		return nil, err
-	}
-
-	apps := make(appsWorkload)
-	for _, w := range ws {
-		if appLabel, ok := w.Labels[cfg.IstioLabels.AppLabelName]; ok {
-			apps[appLabel] = append(apps[appLabel], w)
-		}
-	}
-	return apps, nil
-}
-
```

Comment:
I'm not sure if I like the fact that unused code is removed.  Perhaps it should be commented out with a comment that says it's dead code, so that an author can decide whether it serves a current purpose (like an example) or future purpose (it's use is imminent).  This is true of several removals in the PR.  If a reviewer says it can go, then fine.

Reply:
Fair! On some of the removals, I'm reverting and commenting out instead of deleting.
The ones I left, I think are OK to remove.

Reply:
Is this one falling in such a category, like imminent use? If it stays weeks and weeks as commented / unused code then I think it's better to remove. Pros are to keep file size smaller hence easier to read, and avoid having pieces of (dead) code containing outdated parts and/or unseen bugs that could have potentially been fixed in other (non-dead) parts of the codebase.

Reply:
@israel-hdez The code removal is totally up to you.  Anything of real value should have already been commented out, with a doc note, by the original dev.  Also, we can always get back to it via git.  Not a blocker on my part.  I'm good either way.

Reply:
I removed this one. Also, removed some others with justification in PR comments.